package.json

```diff
@@ -52,7 +52,7 @@
     "lodash.merge": "3.3.2",
     "request": "^2.81.0",
     "streamifier": "0.1.0",
-    "temp": "0.8.1",
+    "temp": "0.9.4",
```

Comment:
The previous version was pinned to a specific version, so I figured I'd keep it there since I don't know if there was a specific reason behind some dependencies being pinned and some not.

Reply:
Looked into it, and it seems like `temp` was pinned when we'd forked it (https://github.com/lever/DocuSign-Node-Client/commit/7b7d11358497274aa0202dd2f4641ed81a4099c0), and the forked repo has since removed its use of `temp` (after switching to a new version of the DocuSign eSign API Node client according to this commit (https://github.com/docusign/docusign-esign-node-client/commit/0ce0b0815f3c78586aa9062e9aaf6004b9312cb6#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519). Sharing for context but in agreement that keeping it the same seems reasonable.

Reply:
In theory we will be able to remove this package dependency completely once all of our customers enable Docusign via OAuth (we use the new Docusign client as-is for that implementation). Unfortunately, the uptake has been long and slow.